hack/helm_vars/wire-server/kind-values.yaml

```diff
@@ -0,0 +1,22 @@
+cassandra-migrations:
+  imagePullPolicy: Never
```

Comment:
what about `spar-migrate-data` and `galley-migrate-data`? Are those missing here for a reason?

Reply:
Just forgot about them, of course things didn't fail because I didn't test with a tag which is not already published :disappointed: